components/draggables/DraggableItem.tsx

```diff
@@ -17,22 +17,41 @@ interface IProps {
   addItem: () => void;
   deleteSet: () => void;
   addItemRef: RefObject<HTMLButtonElement>;
+  isDraggingOver: boolean;
 }
 
-const DraggableItem = ({ itemId, index, setId, addItem, deleteSet, addItemRef }: IProps) => {
-  const { sets, setSets, items, setItems, itemRefs, setItemRefs, focussedId } = useContext(
-    ViewContext,
-  );
+const DraggableItem = ({
+  itemId,
+  index,
+  setId,
+  addItem,
+  deleteSet,
+  addItemRef,
+  isDraggingOver,
+}: IProps) => {
+  const {
+    sets,
+    setSets,
+    items,
+    setItems,
+    itemRefs,
+    setItemRefs,
+    handleRefs,
+    setHandleRefs,
+    focussedId,
+  } = useContext(ViewContext);
   const set = sets[setId];
   const item = items[itemId];
   const itemRef = useRef<HTMLInputElement>();
+  const handleRef = useRef<HTMLInputElement>();
```

Comment:
handle is clearly not an `input`. it's `div`
I wonder why TS doesn't complain about this.

Reply:
htmldivelement?